### DISCUSSION TITLE
how do i use microsd with ruri?

### DISCUSSION THREAD

**Discussion by whyakari** [Moe-hacker/ruri#4] [General]
well, first I want to apologize for asking again (the other time it was with the termux-container)

the question is: can i do the same thing with ruri? just like we did [here](https://github.com/Moe-hacker/termux-container/discussions/27#discussioncomment-4966234) ♡

**Comment by Moe-hacker** [comment]
> well, first I want to apologize for asking again (the other time it was with the termux-container)
> 
> the question is: can i do the same thing with ruri? just like we did [here](https://github.com/Moe-hacker/termux-container/discussions/27#discussioncomment-4966234) ♡

 Sorry that the notification of this discussion has only send to my another Github account,so maybe the reply is late.      
Ruri is just a program like `chroot`, to run a container with basic protections.   You can use it to start a container after creating it, try:
```shell
#mount your microsd
mkdir -p test
mount /dev/block/mmcblk0p1 ./test
if [[ $? != 0 ]];then
  LOOP=$(losetup -f)
  losetup $LOOP /dev/block/mmcblk0p1
  mount $LOOP ./test
fi
#start the daemon
ruri -D
#run the container in unshare mode
ruri -u -w ./test
```
